ndscan/experiment/parameters.py

```diff
@@ -374,3 +417,36 @@ def eval_default(self, get_dataset: GetDataset) -> str:
 
     def make_store(self, identity: Tuple[str, str], value: str) -> StringParamStore:
         return StringParamStore(identity, value)
+
+
+class BoolParam:
+    HandleType = BoolParamHandle
+    StoreType = BoolParamStore
+    CompilerType = TBool
+
+    def __init__(self,
+                 fqn: str,
+                 description: str,
+                 default: bool,
+                 is_scannable: bool = False):
```

Comment:
This should probably default to `True`, as the idea for `is_scannable` is to be able to disable scans for a few parameters where one wants to make absolutely sure they are not scanned by accident, or the rare cases where some code can't handle scans.

Reply:
Sure! I had defaulted it to `False` as the param doesn't currently handle scans, but I'll change that.